src/python/WMCore/MicroService/Unified/Common.py

```diff
@@ -190,7 +190,8 @@ def getPileupSubscriptions(datasets, phedexUrl, group=None, percentMin=99):
         url = "%s/subscriptions?group=%s" % (phedexUrl, group)
         url += "&percent_min=%s&dataset=%s"
     else:
-        url = "%s/subscriptions?percent_min=%s&dataset=%s" % phedexUrl
+        url = "%s/subscriptions?" % phedexUrl
+        url += "percent_min=%s&dataset=%s"
```

Comment:
I see here, you have fixed that error from the previous commit. So sorry for my previous comment then. But it would be better if we squash commits like that - and then only the final version of the code will be reviewed (leaving aside the late review itself, of course - for which I can blame only myself ;) )

Reply:
I agree, Todor. I overlooked the number of commits. It should have been squashed before getting merged.